README.md

```diff
@@ -52,36 +52,267 @@ Our <a target="_blank" href="https://pieces.app">Flagship Desktop App</a> provid
 
 You can find guidance on how to contribute to the Pieces for Developers documentation in the [CONTRIBUTING.md](CONTRIBUTING.md) file.
 
+# Getting Started
+
+This documentation was formulated according to the [Diataxis model](https://diataxis.fr/) to help you quickly clone the repository, start up your local server, and implement new documentation. 
+
+You will find links to relevant information on dependencies as well as an internal MDX style guide and MDX template where necessary. 
+
+## Cloning the Repository
+
+You will need to clone the repository in order to make changes to the documentation. 
+
+First, open your local terminal and ensure you are in the directory you wish to clone this repository into.
+
+Next, paste the following command into your terminal to clone the repository into the active directory:
+
+```
+git clone https://github.com/pieces-app/documentation.git
+```
+:::info 
```

Comment:
Not sure if this will render in Github Readme, since it doesn't behave like Docusaurus